server/src/test/java/com/broadcom/lsp/cobol/usecases/TestSqlIncludeStatementNotDefinedCorrectly.java

```diff
@@ -33,7 +33,7 @@
           + "       DATA DIVISION.\n"
           + "       WORKING-STORAGE SECTION.\n"
           + "       01 {$*SQLCA}.\n"
-          + "       EXEC  {INCLUDE|1} STRUCT1 END-EXEC.";
+          + "       EXEC  INCLUDE STRUCT1 END-EXEC.{|1}";
```

Comment:
Is it correct? How the user will see it?

Reply:
Fixed